xwiki-platform-distribution/xwiki-platform-distribution-ui/xwiki-platform-distribution-ui-base/src/main/resources/XWiki/XWikiPreferences.xml

```diff
@@ -2815,6 +2815,9 @@ Your account on ${wikiname} has been successfully activated. You can now login a
 &lt;meta name="keywords" content="$escapetool.xml($keywords.replaceAll('[|,]', ' ').trim())" /&gt;
 &lt;meta name="rating" content="General" /&gt;
 &lt;meta name="author" content="$escapetool.xml($!xwiki.getUserName($doc.getAuthor(), false))" /&gt;
+#foreach($uix in $services.uix.getExtensions("org.xwiki.platform.head", {'sortByParameter' : 'order'}))
```

Comment:
Since users will see this in the Admin UI, I believe it's very important to have some comments to explain what this is about.

I was hesitating between putting this in the Admin UI field, or in the template file for the header. The advantage of putting it in the Admin UI field, is that it's easier to control if the user doesn't want it vs having to disable UIXs. OTOH, it would be more consistent with other UIXP to have it in the template.

WDYT?